### PR TITLE
Robga/policy

### PR DESCRIPTION
**Pull request by robga**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

This version is to update $filter and $top for collection get of policy artifacts

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

no breaking change

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by robga** [comment]
Please let me know if you agree to remove the ById series in the policyAssignments.json.

**Comment by raych1** [comment]
> Please let me know if you agree to remove the ById series in the policyAssignments.json.

@robga , can you point out the location of `ById` removed? I didn't find it.

File: specification/resources/resource-manager/Microsoft.Authorization/stable/2020-03-01/policyDefinitions.json [hunk 1]
```diff
@@ -0,0 +1,681 @@
...
+        "x-ms-examples": {
+          "List built-in policy definitions": {
+            "$ref": "./examples/listBuiltInPolicyDefinitions.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
```
Comment (pilor) [inline specification/resources/resource-manager/Microsoft.Authorization/stable/2020-03-01/policyDefinitions.json:418]: can you add filter here as well?  Its valuable for category and potentially filtering out the static policyType

**Review by pilor — APPROVED** [review]
1 suggestion

File: specification/resources/resource-manager/Microsoft.Authorization/stable/2020-03-01/policyDefinitions.json (hunk 1 above)
Comment (robga) [reply specification/resources/resource-manager/Microsoft.Authorization/stable/2020-03-01/policyDefinitions.json:418]: added.

**Comment by robga** [comment]
> > Please let me know if you agree to remove the ById series in the policyAssignments.json.
> 
> @robga , can you point out the location of `ById` removed? I didn't find it.

Nevermind, I should not remove those.

**Pull request merged**